students/ZidanLuo/session01/break_me.py

```diff
@@ -0,0 +1,18 @@
+def NameError ():
+    print(x)
+
+# Passing arguments of the wrong type should result in a TypeError
+def TypeError(x):
+    x = x + 1
+
+def SyntaxError():
```

Comment:
I doin't see a syntax error here -- I do see a TypeError though...

A syntax error will get caught when you try to run the code, before it even starts running.